taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/TestTaskAcquisition.java

```diff
@@ -373,6 +384,93 @@ void process_instance_with_multiple_executions_should_result_in_multiple_taskana
     }
   }
 
+  @WithAccessId(
+      user = "teamlead_1",
+      groups = {"taskadmin"})
+  @Test
+  void should_createTaskanaTask_When_SystemConnectorHasCorrectSystemEngineIdentifier()
+      throws Exception {
+
+    final Map<String, SystemConnector> originalSystemConnectors =
+        new HashMap<>(adapterManager.getSystemConnectors());
+
+    setSystemConnector("default");
+
+    String processInstanceId =
+        this.camundaProcessengineRequester.startCamundaProcessAndReturnId(
+            "simple_user_task_process", "");
+    List<String> camundaTaskIds =
+        this.camundaProcessengineRequester.getTaskIdsFromProcessInstanceId(processInstanceId);
+
+    Thread.sleep((long) (this.adapterTaskPollingInterval * 1.2));
+
+    for (String camundaTaskId : camundaTaskIds) {
```

Comment:
isn't there a "smarter" way to assert that? (like containsExactly()... - just asking... ;-))

Reply:
yes, this is a bad copy paste ;)

Reply:
are you going to change that? Or do I simply don't see your change?